projects/canopy/src/lib/variant/variant.stories.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Input } from '@angular/core';
+
+import { select, withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+
+import { LgButtonModule } from '../button';
+import { LgCardModule } from '../card';
+import { LgGridModule } from '../grid';
+import { LgIconModule, LgIconRegistry } from '../icon';
+import { iconsArray } from '../icon/icons.stories';
+import { LgSpacingModule } from '../spacing';
+import { Variant } from './variant.interface';
+import { LgVariantModule } from './variant.module';
+import { notes } from './variant.notes';
+
+const variants = ['generic', 'info', 'success', 'warning', 'error'];
+
+@Component({
+  selector: 'lg-variant-story',
+  template: `
+    <lg-card [lgVariant]="variant" lgPaddingLeft="lg" lgPaddingRight="lg">
+      <lg-card-content>
+        <div lgContainer>
+          <div lgRow>
+            <div [lgCol]="12" [lgColMd]="8">
```

Comment:
These grid directives don't need the `[]`

Reply:
Fixed 👍